free-programming-books-ru.md

```diff
@@ -324,9 +324,12 @@
 
 #### React
 
+* [Изучаем React](https://codernet.ru/books/js/izuchaem_react_kirupa_chinnatambi/) - Кирупа Чиннатамби
```

Comment:
not sure this is an authorized distribution. check the website of the translator/publisher. Please see CONTRIBUTING.md guidelines.

Reply:
@eshellman Thank you for pointing it out, looks like you were right! Adjusted the PR, my bad 😅